blueskyacademy/bluesky-nextjs
# add new icons

## Description
- Add new icons


## Why
- Add new icons


## How to QA
- User can see some icons in gallery, news
- User can see background with different gradients


## Screenshots
<img width="1382" alt="Screen Shot 2022-02-12 at 10 02 56" src="https://user-images.githubusercontent.com/74655110/153694284-b75fd8b1-1359-43d7-a333-4b7291c08f6c.png">
<img width="1392" alt="Screen Shot 2022-02-12 at 10 03 06" src="https://user-images.githubusercontent.com/74655110/153694283-01cb9bbb-0615-43f0-95e5-ac4a223e200d.png">
<img width="1424" alt="Screen Shot 2022-02-12 at 10 03 17" src="https://user-images.githubusercontent.com/74655110/153694285-e3b19fd5-8e43-4f7c-bcb0-65dc90fbf306.png">
